SPECS/graphviz/graphviz.spec

```diff
@@ -265,7 +267,7 @@ sed -i 's|_MY_JAVA_INCLUDES_|-I%{java_home}/include/ -I%{java_home}/include/linu
 
 %configure --with-x --disable-static --disable-dependency-tracking \
 	--without-mylibgd --with-ipsepcola --with-pangocairo \
-	--without-gdk-pixbuf --with-visio --disable-silent-rules \
+	--without-gdk-pixbuf --with-visio --disable-silent-rules --with-freetypeincludedir="%{_includedir}/freetype2" --with-freetypelibdir="%{_libdir}" \
```

Comment:
One more request: could you put the new switches in a new line to keep the lines shorter?